scripts/build_images.sh

```diff
@@ -66,6 +63,6 @@ for VERSION in "${!VERSION_ARCH_MAP[@]}"; do
     done
 
     # Create the manifest list for each version under the same repository
-    MANIFEST_TAG="$DOCKER_REGISTRY/$IMAGE_NAME:$VERSION"
```

Comment:
Don't particularly understand why this worked on GitLab but not on DockerHub, but w/e